x-pack/plugins/security_solution/common/endpoint/schema/trusted_apps.ts

```diff
@@ -6,6 +6,14 @@
 
 import { schema } from '@kbn/config-schema';
 
+export const DeleteTrustedAppsRequestSchema = {
+  params: schema.object({
+    id: schema.string(),
+  }),
+  body: schema.maybe(schema.never()),
```

Comment:
"maybe never".

This is the most passive-aggressive code I've seen in a while :joy:

Reply:
😆 
I'm actually wondering if I should remove this. My intent was to have the schema validate/stop requests with invalid portion of it defined before it reached the handler. Looking around Kibana, does not look like that is normally done. What do you think - should I just remove it?